misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/RangeFormatterTest.java

```diff
@@ -113,26 +105,34 @@ public void testParserResources(String sourcePath) throws IOException, Formatter
      */
     public abstract String getTestResourceDir();
 
+    public abstract String getConfigJsonFileName();
+
     protected Object[][] getConfigsList() {
         if (this.testSubset().length != 0) {
             return this.testSubset();
         }
         List<String> skippedTests = this.skipList();
         try {
-            return Files.walk(this.resourceDirectory.resolve(this.getTestResourceDir()).resolve(ASSERT_DIR))
-                    .filter(path -> {
-                        File file = path.toFile();
-                        return file.isFile() && file.getName().endsWith(".bal")
-                                && !skippedTests.contains(file.getName());
-                    })
-                    .map(path -> new Object[]{path.toFile().getName(), this.getTestResourceDir()})
-                    .toArray(size -> new Object[size][2]);
-        } catch (IOException e) {
+            File jsonConfigFile = Paths.get(resourceDirectory.toString(), this.getTestResourceDir(),
+                    this.getConfigJsonFileName()).toFile();
```

Comment:
As a practice, we don't really need to use `this` for function calls. It's fine to use it for instance variables, to distinguish them with local variables. But since functions doesn't have such a concept, using `this` for function calls can be an overkill.

Reply:
Removed all the `this` for function calls.